src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -114,7 +115,7 @@ public function include_payment_buttons( $file, $name, $template ) {
 	 */
 	public function maybe_create_stripe_payment_intent() {
 
-		if ( ! tribe( Merchant::class )->is_connected() ) {
+		if ( ! tribe( Merchant::class )->is_connected() && tribe( Module::class )->is_checkout_page() ) {
```

Comment:
@moraleida I assume you wanted to skip all other pages but the checkout page, so the logic should be: ` ! tribe( Module::class )->is_checkout_page() `

Otherwise, I am getting this error on all pages:
```[08-Feb-2022 10:30:55 UTC] PHP Notice:  Undefined property: stdClass::$code in /app/public/wp-content/plugins/event-tickets/src/Tickets/Commerce/Gateways/Stripe/Client.php on line 562
[08-Feb-2022 10:30:55 UTC] PHP Stack trace:
[08-Feb-2022 10:30:55 UTC] PHP   1. {main}() /app/public/index.php:0
[08-Feb-2022 10:30:55 UTC] PHP   2. require() /app/public/index.php:17
[08-Feb-2022 10:30:55 UTC] PHP   3. wp() /app/public/wp-blog-header.php:16
[08-Feb-2022 10:30:55 UTC] PHP   4. WP->main() /app/public/wp-includes/functions.php:1310
[08-Feb-2022 10:30:55 UTC] PHP   5. do_action_ref_array() /app/public/wp-includes/class-wp.php:771
[08-Feb-2022 10:30:55 UTC] PHP   6. WP_Hook->do_action() /app/public/wp-includes/plugin.php:522
[08-Feb-2022 10:30:55 UTC] PHP   7. WP_Hook->apply_filters() /app/public/wp-includes/class-wp-hook.php:331
[08-Feb-2022 10:30:55 UTC] PHP   8. TEC\Tickets\Commerce\Gateways\Stripe\Hooks->maybe_create_stripe_payment_intent() /app/public/wp-includes/class-wp-hook.php:307
[08-Feb-2022 10:30:55 UTC] PHP   9. TEC\Tickets\Commerce\Gateways\Stripe\Client->create_payment_intent() /app/public/wp-content/plugins/event-tickets/src/Tickets/Commerce/Gateways/Stripe/Hooks.php:108
[08-Feb-2022 10:30:55 UTC] PHP  10. TEC\Tickets\Commerce\Gateways\Stripe\Client->post() /app/public/wp-content/plugins/event-tickets/src/Tickets/Commerce/Gateways/Stripe/Client.php:143
[08-Feb-2022 10:30:55 UTC] PHP  11. TEC\Tickets\Commerce\Gateways\Stripe\Client->request() /app/public/wp-content/plugins/event-tickets/src/Tickets/Commerce/Gateways/Stripe/Client.php:480
[08-Feb-2022 10:30:55 UTC] PHP  12. TEC\Tickets\Commerce\Gateways\Stripe\Client->prepare_errors_to_display() /app/public/wp-content/plugins/event-tickets/src/Tickets/Commerce/Gateways/Stripe/Client.php:413```